modules/rtm/R/fortran.datamodule.R

```diff
@@ -22,14 +22,16 @@
 #'       z <- seq(exp(1), pi, length.out=42)
 #'       l <- list(x=x, y=y, z=z) ## NOTE that names must be explicitly declared
 #'       l.types <- c('real','integer', 'real*4', 'real*8')
-#'       fortran_data_module(l, l.types, 'testmod')
-#'  
+#'       fortran_data_module(l, l.types, 'testmod',
+#'         file.path(tempdir(), "testmod.f90"))
+#'
```

Comment:
cc @ashiklom for 👀 👍 / 👎 -- idea is just to convince R CMD check that examples don't write any files to userland.

Reply:
👍 This code is ancient and should probably be moved to ancillary scripts at best. But if this makes R CMD check happy, then I'm happy.